concepts/basics/introduction.md

```diff
@@ -0,0 +1,83 @@
+# Introduction
+
+To start the C++ journey we are starting with variables, function calls, and comments.
+
+## Variables
+
+C++ is a typed language.
+The type of a variable has to be stated upon the declaration of a variable.
+Its type cannot change during its lifetime.
```

Comment:
Should I drop the rest after "Its type cannot change"? It would still be correct, but I would avoid to explain scopes and lifetimes and memory management? @siebenschlaefer

Reply:
Lifetimes probably deserve their own concept node. I'm not sure what to do with scopes, but I am still attracted to postponing their discussion.

The statement that variables' types cannot change only makes sense in the context of other languages in which something like that _seems_ to be possible (in dynamic languages _values_ may have types; in static languages _variables_ have types). Therefore perhaps put this discussion in an `exercism/note` block?